### PR TITLE
New release process using docker, circleci and a centralized building tool

### PR DESCRIPTION
**Pull request by sdurrheimer**
This will help for #21

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.build/
+.tarballs/
+
+!.build/linux-amd64/
```
Comment (grobie) [inline .dockerignore]: Please end all files on a newline.

Comment (grobie) [inline .dockerignore]: Same below in `.promu.yml`.

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,6 @@
-.build
-consul_exporter
+/.build
+/.release
+/.tarballs
+/consul_exporter
 *.tar.gz
 *-stamp
```
Comment (grobie) [inline .gitignore]: The stamps are not used anymore now.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+sudo: false
+
+language: go
+
+go:
+- 1.5.4
```
Comment (grobie) [inline .travis.yml]: Not sure we need that.

**Comment by grobie** [comment]
:+1: after fixing the newlines.

File: .travis.yml (hunk 3 above)
Comment (sdurrheimer) [inline .travis.yml]: People might still build it with 1.5, we do the same with other components.
Moreover, travis builds are fast, so it doesn't cost much time.

**Comment by sdurrheimer** [comment]
Done

**Pull request merged**